packages/react/src/components/radio-card-group/styled-components.ts

```diff
@@ -0,0 +1,149 @@
+import styled from 'styled-components';
+import { Theme } from '../../themes';
+import { focus } from '../../utils/css-state';
+
+interface InputContainerProps {
+    disabled?: boolean;
+    isMobile: boolean;
+    isChecked?: boolean;
+    theme: Theme;
+}
+
+interface DescriptionProps {
+    disabled?: boolean;
+    isMobile: boolean;
+    isChecked?: boolean;
+    theme: Theme;
+}
+
+interface CardProps {
+    disabled?: boolean;
+    isMobile: boolean;
+    isChecked?: boolean;
+    theme: Theme;
+}
+
+export const RadioInput = styled.span<CardProps>`
+    background-color: ${({ disabled, theme }) => (disabled ? theme.greys['light-grey'] : theme.greys.white)};
+    border: 1px solid ${({ disabled, theme }) => (disabled ? theme.greys.grey : theme.greys['dark-grey'])};
+    border-radius: 50%;
+    box-sizing: border-box;
+    display: inline-block;
+    height: ${({ isMobile }) => (isMobile ? 'var(--size-1halfx)' : 'var(--size-1x)')};
+    margin-right: var(--spacing-2x);
+    vertical-align: middle;
+    width: ${({ isMobile }) => (isMobile ? 'var(--size-1halfx)' : 'var(--size-1x)')};
+`;
+
+function getContentColor({ disabled, isChecked, theme }: InputContainerProps): string {
+    if (disabled) {
+        return theme.greys['mid-grey'];
+    } if (isChecked) {
+        return theme.main['primary-2'];
+    }
+    return theme.greys['dark-grey'];
+}
+
+function getLabelBackgroundColor({ disabled, isChecked, theme }: CardProps): string {
+    if (disabled) {
+        return theme.greys['light-grey'];
+    } if (isChecked) {
+        return theme.main['primary-1.4'];
+    }
+    return theme.greys.white;
+}
+
+function getLabelBorderColor({ disabled, isChecked, theme }: CardProps): string {
+    if (disabled) {
+        return theme.greys.grey;
+    } if (isChecked) {
+        return theme.main['primary-2'];
+    }
+    return theme.greys['dark-grey'];
+}
+
+export const Fieldset = styled.fieldset`
+  border: none;
```

Comment:
C'est plate parce qu'il y a un problème dans stylelint qui nous empêche de valider l'indentation, mais ça devrait être 4 spaces partout. Il y a plusieurs components dans ce fichier qui ont seulement 2 spaces 🫤
```suggestion
    border: none;
```